docs/usage/faq.md

```diff
@@ -0,0 +1,49 @@
+# Frequently asked questions
+
+## Why should I use tuist?
```

Comment:
👍 

Managing Xcode projects is hard, especially once you go beyond a few developers working on a simple project with a handful of targets to larger team working on a workspace with several projects, each of which have several targets.

Managing large projects using the UI interface is error prone, and not very reproducible. Having a project definition in code or configuration helps make this more manageable.  This is actually quite common in other places too e.g. Swift PM for swift package, gradle for Android, cmake for c++ project etc...